integration-cli/docker_cli_login_test.go

```diff
@@ -14,8 +15,7 @@ func (s *DockerSuite) TestLoginWithoutTTY(c *check.C) {
 	cmd.Stdin = bytes.NewBufferString("buffer test string \n")
 
 	// run the command and block until it's done
-	if err := cmd.Run(); err == nil {
-		c.Fatal("Expected non nil err when loginning in & TTY not available")
-	}
+	err := cmd.Run()
+	c.Assert(err, checker.NotNil, check.Commentf("Expected non nil err when loginning in & TTY not available"))
```

Comment:
this commentf can be a regular comment on the same line or the line previous.
`// Expected non nil err when loginning in & TTY not available`
It contains no variables to be interpolated.